### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by sub1120**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature-template.md [hunk 1]
```diff
@@ -1,19 +1,20 @@
 ---
-name: Feature Template
-about: For any new feature
-title: "[FEAT]: Short title"
+name: Feature request
+about: Suggest an idea for this project
+title: "[FEAT] : Short title"
```
Comment (shivamvijaywargi) [inline .github/ISSUE_TEMPLATE/feature-template.md]: I don't think space is required between "]" and ":"

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
 ---
 name: Bug report
 about: Create a report to help us improve
-title: "[BUG]: Short title"
+title: "[BUG] : Short Title"
```
Comment (shivamvijaywargi) [inline .github/ISSUE_TEMPLATE/bug-report.md]: I don't think space is required between "]" and ":"

**Review by shivamvijaywargi — CHANGES_REQUESTED** [review]
Please check the comments

File: .github/ISSUE_TEMPLATE/feature-template.md (hunk 1 above)
Comment (sub1120) [reply .github/ISSUE_TEMPLATE/feature-template.md]: space removed

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 2 above)
Comment (sub1120) [reply .github/ISSUE_TEMPLATE/bug-report.md]: space removed

**Review by shivamvijaywargi — APPROVED** [review]

**Pull request merged**